generated/proto/policy.proto

```diff
@@ -34,3 +34,13 @@ message Policy {
 	Resolution resolution = 1;
 	Retention retention = 2;
 }
+
+message CompressedPolicy {
+	Policy policy = 1;
+	int64 id = 2;
+}
+
+message ActivePolicies {
+	repeated CompressedPolicy compressedPolicies = 1;
```

Comment:
The convention is to use underscore for field names in protobuf definitions: https://developers.google.com/protocol-buffers/docs/style